src/s1reader/s1_orbit.py

```diff
@@ -255,6 +257,53 @@ def get_orbit_file_from_dir(zip_path: str, orbit_dir: str, auto_download: bool =
 
     # search for orbit file
     orbit_file_list = glob.glob(os.path.join(orbit_dir, 'S1*.EOF'))
+
+    orbit_file = get_orbit_file_from_list(zip_path, orbit_file_list)
+
+    if orbit_file:
+        return orbit_file
+
+    if not auto_download:
+        msg = (f'No orbit file was found for {os.path.basename(zip_path)} '
+                f'from the directory provided: {orbit_dir}')
+        warnings.warn(msg)
```

Comment:
```suggestion
        raise FileNotFoundError(msg)
```
What do we think about raising an error instead of issuing a warning? An orbit is essential to processing no?

Reply:
Makes sense to me, but just did not want to change the way that the current version of the function handles the missing orbit file (see below).
I'm open to changing the warning to Error. @vbrancat @hfattahi Any thoughts?

https://github.com/opera-adt/s1-reader/blob/d8f9e80364c02ef338062525065b1122599075d6/src/s1reader/s1_orbit.py#L266-L270